samples/ownerdrw/ownerdrw.cpp

```diff
@@ -345,6 +345,11 @@ OwnerDrawnFrame::OwnerDrawnFrame(wxFrame *frame, const wxString& title,
         {
             pListBox->GetItem(ui)->SetBackgroundColour(wxColor(0, 0, 0));
         }
+        else
+        if ( ui == 5 ) // black on dark gray is barely visible
+        {
+            pListBox->GetItem(ui)->SetBackgroundColour(*wxYELLOW);
+        }
```

Comment:
This indentation seems strange, let's write it like this
```suggestion
        else if ( ui == 5 ) // black on dark gray is barely visible
        {
            pListBox->GetItem(ui)->SetBackgroundColour(*wxYELLOW);
        }
```

Reply:
Done.